CIP-1694/README.md

```diff
@@ -0,0 +1,1161 @@
+
+The term limit will reset whenever the  "New constitutional committee and/or threshold" governance action is enacted, even if the same committee is re-elected
+and the threshold remains unchanged.  This allows Ada holders to confirm their confidence in the committee if they wish.
+Note that the term limit is calculated at the point the committee is elected.  Any change in the underlying protocol
+parameter only affects the term that applies to future committees, and does not change the term of the current committee.
+
+
+<!--------------------------- Constitutional committee ------------------------>
+<!---------------------------           DReps          ------------------------>
+
+### Delegated representatives (DReps)
+
+> **Warning**
+> CIP-1694 DReps **should not be conflated** with Project Catalyst DReps.
+
+<!-- TODO find another name that still points to liquid democracy. -->
+
+#### Pre-defined DReps
+
+In order to participate in governance, each stake credential must be delegated to a DRep.
+Ada holders will generally delegate their voting rights to a registered DRep
+that will vote on their behalf.  In addition, two pre-defined DRep options are available:
+
+* `Abstain`
+
+  If an Ada holder delegates to `Abstain`, then their stake is actively marked
+  as not participating in governance.
+
+  The effect of delegating to `Abstain` on chain is that the delegated stake *will not* be considered to be
+  a part of the active voting stake.  However, the stake *will* be considered to be registered for the
+  purpose of the incentives that are described in [Incentives for Ada holders to delegate voting stake](#incentives-for-ada-holders-to-delegate-voting-stake).
+
+* `No Confidence`
+
+  If an Ada holder delegates to `No Confidence`, then their stake is counted as
+  a **no** vote on every governance action apart from a "Motion of no confidence".
+  This also signals that they have no confidence in the existing constitutional committee.
+
+  The effect of delegating to `No Confidence` on chain is that this stake *will* be considered to be
+  a part of the active voting stake. It will count as a `Yes` vote on every `No Confidence`
+  action and a `No` vote on every other action.
+  It also serves as a directly auditable measure of the confidence of Ada holders in the constitutional
+  committee.
+
+
+> **Note**
+> Any Ada holder may register their stake credential as a DRep and delegate to themselves if they wish to actively participate in
+> voting.
+
+#### Registered DReps
+
+In Voltaire, existing stake credentials will be
+able to delegate their stake to registered DReps for voting purposes,
+in addition to the current delegation to stake pools for block production.
+DRep delegation will mimic the existing stake delegation mechanisms (via on-chain certificates).
+Similarly, DRep registration will mimic the existing stake registration mechanisms.
+Additionally, registered DReps will need to vote regularly to still be considered active.
```

Comment:
> Additionally, registered DReps will need to vote regularly to still be considered active.

Has there already been a discussion about what "regularly" could mean?

Reply:
not yet